tests/unit/data/interAttendanceSheet.test.js

```diff
@@ -0,0 +1,179 @@
+const sinon = require('sinon');
+const expect = require('expect');
+const FileHelper = require('../../../src/helpers/file');
+const InterAttendanceSheet = require('../../../src/data/interAttendanceSheet');
+
+describe('getPdfContent', () => {
+  let downloadImages;
+
+  beforeEach(() => {
+    downloadImages = sinon.stub(FileHelper, 'downloadImages');
+  });
+
+  afterEach(() => {
+    downloadImages.restore();
+  });
+
+  it('jsp', async () => {
```

Comment:
Il faudrait mettre un it un peu plus explicite ^^

Reply:
oupsi :joy: